src/bundle/ui-dev/src/modules/universal-discovery/components/breadcrumbs/breadcrumbs.js

```diff
@@ -41,6 +42,10 @@ const Breadcrumbs = () => {
     const toggleHiddenListVisible = useCallback(() => {
         setHiddenListVisible(!hiddenListVisible);
     }, [setHiddenListVisible, hiddenListVisible]);
+    const handleTogglerClick = (event) => {
+        event.stopPropagation();
```

Comment:
Note: we need to stop this propagation because otherwise it may be caught again in `hideHiddenMenuOnClickOutside`. Also, note that this event is a react event, which may have a different propagation than a standard DOM event.